build_tools/cmake/build_host_and_riscv.sh

```diff
@@ -72,6 +72,7 @@ args=(
   -DIREE_ENABLE_ASSERTIONS=ON
   -DIREE_BUILD_COMPILER=OFF
   -DIREE_BUILD_SAMPLES=ON
+  -DIREE_ENABLE_CPUINFO=OFF
```

Comment:
I think this file is actually no longer used and can be deleted: https://cs.github.com/iree-org/iree?q=build_host_and_riscv.sh It's cleanup after turning down Kokoro that I haven't gotten to yet

Reply:
Actually, I need to use this script to test the build locally, because either `build_all.sh` or `build_host.sh` has CUDA and/or PYBINDING enabled. I _guess_ we want to trim the flags in `build_host.sh` to not build for every backend, but that's out of scope from this PR.

Reply:
Building tools with the Cuda backend included should work everywhere now. You don't need to have a GPU or cuda installed. If the deps aren't available it fetches them on demand

Reply:
You are right, I don't know why my first try of running `build_host_tools.sh` failed.

There are other legacy cross-compile scripts. Probably should clean all of them in a separate PR.